cache/noop-cache/src/main/java/org/jdbi/v3/cache/noop/NoopCache.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
```

Comment:
This implementation has no dependencies. Can it live in core, or is it really worth its own module? It is not literally a single class, but it is close, and I am not sure we want to be so modular that we break every little feature out.

Reply:
We can. Good idea. I will move that into the core.

Reply:
on second thought, this prevents the service loader from working or make the module mandatory. I see the point of feature proliferation though.

Reply:
I feel strongly about all our modules being able to be loaded through the service loader. Moving this into the core would prevent that for the NoopCache. I leave it as a module, this is intended for debugging/testing. The jar has 6k, so it would increase the size of the core jar by ~ 1%.